runbotics/runbotics-actions-windows/src/automation/office/excel.action-handler.ts

```diff
@@ -119,6 +121,33 @@ export default class ExcelActionHandler extends StatefulActionHandler {
         return rowCounter;
     }
 
+    async insertColumnsBefore(input: ExcelInsertColumnsInput): Promise<void> {
+        const startingSheet = this.session.ActiveSheet.Name;
+        const workingWorksheet = this.session.Worksheets(
+            input?.worksheet ?? startingSheet
+        );
+
+        try {
+            const column = this.getColumnCoordinate(input.column);
+            const amount = input.amount;
+
+            workingWorksheet
+                .Range(
+                    workingWorksheet.Columns(column),
+                    workingWorksheet.Columns(column + amount - 1)
+                )
+                .Insert();
+        } catch (e) {
+            throw new Error(
+                `Column has to be a name of the column, e.g. C or column number, e.g. 3
+                Amount has to be a whole positive number, eg. 5.
+                `
+            );
+        }
+
+        this.session.Worksheets(startingSheet);
```

Comment:
were not using this.session.Worksheets(<someWorksheet>).ACTIVATE() so, we're not switching worksheets.

this is redundant

Reply:
remove it